screens/Map.js

```diff
@@ -1,31 +1,92 @@
 import React from 'react';
 import {
-  Image,
-  Platform,
-  ScrollView,
-  StyleSheet,
-  Text,
-  TouchableOpacity,
-  View,
-} from 'react-native';
-import { WebBrowser, MapView, Marker } from 'expo';
+  MapView,
+  Location,
+  Permissions,
+} from 'expo';
+
+import data from '../data.js'
 
 export default class Map extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      markers: [],
+      followsUserLocation: true,
+      mapRegion: {
+        latitude: 40.673868,
+        longitude: -73.970089,
+        latitudeDelta: 0.0922,
+        longitudeDelta: 0.0421,
+      },
+      locationResult: ""
+    };
+  };
+
   static navigationOptions = {
     header: null,
   };
 
+  componentDidMount = async () => {
+    this.setState({
+      markers: data
+    });
+    await this._getLocationAsync();
+  };
+
+  _getLocationAsync = async () => {
+    let {
+      status
+    } = await Permissions.askAsync(Permissions.LOCATION);
```

Comment:
what does this do? does it give us that locations permissions pop up or do we have to handle that ourselves?

Reply:
We need this to access the iOS user's location. It does give a location permission pop up to the user, saying that the app is asking for permission that comes from the Expo Permissions API.

When we deploy to the Apple Store though, Expo suggests "you should consider adding additional metadata to your app in order to customize the system permissions dialog and explain why your app requires permissions"

Reply:
neat! that sounds great.